cabal-install-parsers/cabal-install-parsers.cabal

```diff
@@ -114,7 +114,7 @@ test-suite cabal-parsers-golden
 
   -- dependencies needing explicit constraints
   build-depends:
-    , ansi-terminal  ^>=0.10
+    , ansi-terminal  >=0.10   && <0.12
```

Comment:
We like `^>=` syntax, so `^>=0.10 || ^>=0.11`

Reply:
Updated.